sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -654,6 +655,7 @@ export interface BlobGetPropertiesHeaders {
     isIncrementalCopy?: boolean;
     isSealed?: boolean;
     isServerEncrypted?: boolean;
+    lastAccessed?: Date;
```

Comment:
lastAccessedOn? We seem to use the `on` suffix except for `lastModified` and presumably other headers...

Reply:
.NET is using `lastAccessed` for the reponse header in Get Blob Properties and Download Blob and `lastAcessedOn` for that in the response body of List Blobs. 
JS followed .NET and I think it's consistent with what we discussed.

Python is a little bit different, they have a `BlobProperties` wrapper which is returnd for all the three operations. While .NET and JS use the underlying headers or fileld directly.